src/Features/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -17719,6 +17719,27 @@ public void Property_ExpressionBody_Update()
         ]);
     }
 
+    [Fact]
+    public void Property_ExpressionBody_Update_Auto_FieldAccess()
+    {
+        var src1 = "class C { int P => 1; }";
+        var src2 = "class C { int P => field; }";
+
+        var edits = GetTopEdits(src1, src2);
+
+        edits.VerifyEdits(
+            "Update [int P => 1;]@10 -> [int P => field;]@10",
+            "Update [=> 1]@16 -> [=> field]@16");
+
+        edits.VerifySemantics(
+            [SemanticEdit(SemanticEditKind.Update, c => c.GetMember<IPropertySymbol>("C.P").GetMethod)],
```

Comment:
I can't remember, should this also have a field add in the list of edits? Or is that bit automatic for properties?

Perhaps a test in https://github.com/dotnet/roslyn/blob/main/src/Compilers/CSharp/Test/Emit2/Emit/EditAndContinue/EditAndContinueTests.cs would be a good idea, to make sure the right metadata is emitted.

Reply:
Should this also have a field add in the list of edits

I don't believe so. At least, I don't see an insert for the backing field in other auto-property tests. The insert of the field is implicitly tested by verifying the diagnostics below when the AddInstanceFieldToExistingType capability isn't specified. I'll take a look at adding a test or two on the compiler side as well.

Reply:
> The insert of the field is implicitly tested by verifying the diagnostics below when the AddInstanceFieldToExistingType capability isn't specified.

I missed this, that's a very good point.

Reply:
Added compiler layer tests

Reply:
We don't need the field insert. We just need to tell the compiler to emit the property and it will emit all symbols it needs to do so.